src/styles/components/_youtube.scss

```diff
@@ -5,6 +5,7 @@
 //
 // =============================================================================
 
+.youtube,
 .w-youtube {
```

Comment:
Should we drop `w-youtube` since the shared code doesn't generate that?

Reply:
It was used for the livestream player, but ill modify that.